code/datums/components/mood.dm

```diff
@@ -199,6 +199,12 @@
 		if(prob(0.05))
 			add_event(null, "jolly", /datum/mood_event/jolly)
 			clear_event(null, "depression")
+	if(owner.has_trait(TRAIT_WEALTHY))
+		if(prob(1))
```

Comment:
lol srsly mate

Reply:
whats wrong

Reply:
-20 mood swing is left to chance is a big boo, especially when it has a conditional. you should remove the timeout on the mood event and have it toggle on when you become sad enough